content/docs/command-reference/plots/index.md

```diff
@@ -65,7 +65,10 @@ epoch, AUC, loss
 ```
 
 In hierarchical file formats (JSON or YAML), an array of consistent objects is
-expected: every object should have the same structure.
+expected: every object should have the same structure. However, if an object has
+extra elements, it won't affect the plot as long as
+[fields specified](#dvc-template-anchors) for X and Y axis are present in all
+the objects.
```

Comment:
OK we have all the info. in but this got too long and complex. Instead of just appending a bunch of conditions to the previous text ("array of consistent ... same structure"), which is incorrect, can we just rewrite it?

Reply:
Pushed new updates.